src/source/beamerouterthemesjtubeamer.dtx

```diff
@@ -91,10 +87,7 @@
 %
 % \subsubsection{Bottombar}
 %
-%   Clear the original definition of sidebar first. Then append the page info to the footline, which could avoid collision on footnote.
-%    \begin{macrocode} 
-\setbeamertemplate{sidebar right}{}
```

Comment:
It turns out to be not the issue of logo color system. Sidebar right holds the placeholder of the logo originally, but here I cleared it like it is in SJTUBeamermin. It will be refactored in the future to make the logo in a different position.

I think the bottom is not the ideal position for a logo.